test/envs/example_test.py

```diff
@@ -7,11 +7,17 @@
 ENV = os.path.join("envs", "example.yaml")
 
 
-def test_load():
+def create_addons(tmpdir):
+    tmpdir.mkdir("example")
+
+
+def test_load(tmpdir):
+    create_addons(tmpdir)
```

Comment:
Not needed for real env files tests since they never have non-existing addons. The test protects from adding such addon.